easybuild/easyconfigs/i/IQ-TREE/IQ-TREE-1.5.6-goolf-1.7.20-MPI.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = "CMakeMake"
+
+name = 'IQ-TREE'
+version = '1.5.6'
+versionsuffix = '-MPI'
```

Comment:
@pescobar Use `-mpi` to match with existing `IQ-TREE` easyconfig that uses `-omp-mpi` for hybrid build?

Also, is there a point to build a separate MPI vs multithreaded build, as opposed to just enabled both `omp` and `mpi` in a single build? Cfr. #3695